compositor_render/src/renderer/format/rgba_to_yuv.wgsl

```diff
@@ -6,6 +6,7 @@ struct VertexInput {
 struct VertexOutput {
     @builtin(position) position: vec4<f32>,
     @location(0) tex_coords: vec2<f32>,
+    @location(1) @interpolate(flat) video_id: i32,
```

Comment:
why do we need video_id in color converter, it should have exactly one input

Reply:
I forgot to remove it :)